tools/internal_ci/linux/grpc_bazel_on_foundry_dbg.sh

```diff
@@ -50,7 +50,7 @@ source tools/internal_ci/helper_scripts/prepare_build_linux_rc
   --strategy=Closure=remote  \
   --genrule_strategy=remote  \
   --experimental_strict_action_env=true \
-  --experimental_remote_platform_override='properties:{name:"container-image" value:"docker://gcr.io/asci-toolchain/nosla-debian8-clang-fl@sha256:496193842f61c9494be68bd624e47c74d706cabf19a693c4653ffe96a97e43e3" }' \
+  --experimental_remote_platform_override="properties:{name:\"container-image\" value:\"docker://$GRPC_FOUNDRY_TESTS_DOCKER_IMAGE\" }" \
```

Comment:
Is there a better approach? I don't feel like maintaining a customized RBE docker image. If this is the only way to go, it's better to talk with the Foundry team to see if they can build it on their side.

Reply:
Plus does this work for a local bazel run?